src/v2/view-builder/views/EnrollProfileUpdateView.js

```diff
@@ -51,5 +51,12 @@ const Footer = BaseFooter.extend({
 
 export default BaseView.extend({
   Body,
-  Footer
+  Footer,
+
+  postRender() {
+    BaseView.prototype.postRender.apply(this, arguments);
+
+    // As per requirement of this flow set secondEmail default to empty string
+    this.model.set('userProfile.secondEmail', '');
```

Comment:
if no input on the secondary email attribute from profile enrollment flow, and we click the finish/submit button, then we set an empty string for secondEmail key, correct?

Reply:
It's always set to empty first.. if user input something and then click finish, then input value would be used.